crates/bitwarden-wasm-internal/src/crypto.rs

```diff
@@ -30,4 +33,15 @@ impl ClientCrypto {
     pub async fn initialize_org_crypto(&self, req: InitOrgCryptoRequest) -> Result<()> {
         Ok(self.0.crypto().initialize_org_crypto(req).await?)
     }
+
+    pub fn make_key_pair(&self) -> Result<MakeKeyPairResponse> {
```

Comment:
We might want to add a couple of code comments to these functions, these are our public API with the clients.